plugin/pkg/admission/wmftoolsenforcer/admission.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
I know your present plan is not to merge this PR, but I am going to review it as if we were going to merge it, and if it happens that we converge on something we want to merge, so much the better.  And if you don't have time to reply to all comments, that is fine too.